sdks/python/apache_beam/io/gcp/bigquery_tools.py

```diff
@@ -558,6 +559,19 @@ def _insert_load_job(
         ))
     return self._start_job(request, stream=source_stream).jobReference
 
+  @staticmethod
+  def _parse_location_from_exc(content, job_id):
+    """Parse job location from Exception content."""
+    if isinstance(content, bytes):
+      content = content.decode('ascii', 'replace')
+      # search for "Already Exists: Job <project-id>:<location>.<job id>"
+      m = re.search(r"Already Exists: Job \S+\:(\S+)\." + job_id, content)
+      if not m:
+        _LOGGER.warning(
+            "Not able to parse BigQuery load job location for {}", job_id)
+        return None
+      return m.group(1)
```

Comment:
can we also return None when if instance is not true and log the message?

Reply:
It was wong indent, now fixed. Thanks for catching this.